src/store/hotspots/hotspotsSlice.ts

```diff
@@ -30,9 +38,8 @@ export type HotspotsSliceState = {
   followedHotspotsObj: Record<string, Hotspot>
   followedHotspots: Hotspot[]
   order: HotspotSort
-  rewards?: Rewards
+  rewards: Rewards
```

Comment:
Would it be beneficial to use a cache record here like we do with the accountRewards? I guess its also caching in wallet API?

Reply:
You're probably right, but it's only one api call at a time and there's caching on the wallet api, so for now I say we just leave it.